src/Makefile.am

```diff
@@ -512,7 +511,6 @@ libqueso_include_HEADERS += contrib/getpot/getpot.h
 
 nodist_libqueso_include_HEADERS =
 nodist_libqueso_include_HEADERS += $(top_builddir)/config_queso.h
-nodist_libqueso_include_HEADERS += $(top_builddir)/src/core/inc/queso.h
```

Comment:
Yeah not sure I get that either.  Autotools is ancient history for me, but I'm guessing that libqueso_include_HEADERS=.../queso.h causes it to get packaged while nodist_libqueso_include_HEADERS says to install, but not package it.  I would guess these two options augment each other, rather than override each other, so the net effect is queso.h ends up both packaged in distributions and installed.  In constrast config_queso.h is only called out as nodist, so it would get installed, but not packaged.